gallery/__init__.py

```diff
@@ -970,21 +959,19 @@ def render_dir(dir_id: int, auth_dict: Optional[Dict[str, Any]] = None):
                            description=description,
                            display_description=display_description,
                            auth_dict=auth_dict,
-                           lockdown=gallery_lockdown,
-                           member_lock=member_lock)
+                           lockdown=gallery_lockdown)
 
 
 @app.route("/view/file/<int:file_id>")
 @auth.oidc_auth('default')
 @gallery_auth
 def render_file(file_id: int, auth_dict: Optional[Dict[str, Any]] = None):
-    member_lock = session['userinfo'].get('member-lock')
-    print(session)
-
     file_model = File.query.filter(File.id == file_id).first()
     if file_model is None:
         abort(404)
-    if file_model.hidden and ((not auth_dict['is_eboard'] and not auth_dict['is_rtp'] and not auth_dict['is_alumni']) or member_lock):
+    if file_model.hidden and ((not auth_dict['is_eboard']
+                               and not auth_dict['is_rtp']
+                               and not auth_dict['is_alumni']) or session['userinfo'].get('member-lock')):
```

Comment:
should this be `.get('member-lock', False)` for the default?

Reply:
False and None to work the same in this context